### PR TITLE
UI Test

### PR DESCRIPTION
**Pull request by wmathurin**
A singe test at this point  (a POC really)
The test does the following (through the UI):
- switch to sandbox
- logs in
- create three records 
- search for each record and open the detail screen for each one
- logs out

### CODE REVIEW

File: shared/test/test/SalesforceTestCase.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        XCUIApplication().launch()
+        loginThroughUI()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+    }
+    
+    func loginThroughUI() {
+        loginDelegate.loginToSalesforce("wolf@cs1.mobilesdk.ee.org", password:"test123456", host:Host.sandbox)
```
Comment (wmathurin) [inline shared/test/test/SalesforceTestCase.swift]: I will move this out to a file similar to test_credentials.json

File: shared/test/pagehelpers/login/LoginHelper.swift [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
+WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+import Foundation
+import XCTest
+
+enum Host {
+    case production
```
Comment (bhariharan) [inline shared/test/pagehelpers/login/LoginHelper.swift]: Should we throw in a custom host here?

File: shared/test/test/SalesforceTestCase.swift (hunk 1 above)
Comment (bhariharan) [inline shared/test/test/SalesforceTestCase.swift]: Let's use the standard test user. At least we won't be exposing admin privileges ;-)

**Comment by bhariharan** [comment]
Looks good to me otherwise. :+1:

File: shared/test/test/SalesforceTestCase.swift (hunk 1 above)
Comment (wmathurin) [inline shared/test/test/SalesforceTestCase.swift]: I can't remember the username / password. Could you email them to me?

Comment (bhariharan) [inline shared/test/test/SalesforceTestCase.swift]: I sent a reset password to your e-mail from that account.

**Pull request merged**